docs/en/operations/configuration-files.md

```diff
@@ -102,6 +102,23 @@ Example:
 961F000000040000000000EEDDEF4F453CFE6457C4234BD7C09258BD651D85
 ```
 
+## Hiding Configuration {#hidden}
```

Comment:
I think this section conceptually belongs into the previous section about encryption. What about
- renaming the caption at l. 68 to "Encrypting and Hiding Configuration"
- explaining in l. 104 that even with encryption, the configuration file after preprocessing will contain the cleartext element. In case this is a problem, we suggest two alternatives: either set file permissions of the preprocessed file to 600 or use the hidden attribute.
- then explain what the hidden attribute does using the same example as for encryption (you kind of do that already)